src/index.html

```diff
@@ -89,6 +89,14 @@ <h2 class="card-title">Placeholder Title!</h2>
             </div>
         </div>
     </div>
+    <div class="divider mb-0 h-0"></div>
+    <!-- Footer -->
+    <footer class="footer footer-center bg-base-200 text-base-content p-4">
+        <aside>
+            <p class="text-wrap w-2/3 text-balance mb-3">For any questions, or to request an invite to the official Tea Game Discord server, please reach out to me at <a class="mailtoui link" href="mailto:teatimedeveloper@gmail.com?subject=Tea Time Discord Access&body=Hello! I would like to request a link to join the Tea Time discord server.">teatimedeveloper@gmail.com</a></p>
```

Comment:
Do you want to link to your own portfolio or add your name as website developer?

Reply:
Good thinking!